### PR TITLE
Update close-master-pr Workflow's Denial Message

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

Didn't intend for this branch to be made on this repo.
Improves the "close PRs from master" workflow's denial message.

### CODE REVIEW

**Review by DangerRevolution — DISMISSED** [review]
this should say Einstein's Engine

File: .github/workflows/close-master-pr.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:    
     - uses: superbrothers/close-pull-request@v3
       with:
-        comment: "Thank you for contributing to the Space Station 14 repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [our git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html) \n\n You can move your current work from the master branch to another branch by doing `git branch <branch_name` and resetting the master branch."
+        comment: "Thank you for contributing to our repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [our git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html). \n\n You can move your current work from the master branch to another branch by [branching](https://git-scm.com/docs/git-branch) from and [resetting](https://git-scm.com/docs/git-reset) the master branch."
```
Comment (Pspritechologist) [inline .github/workflows/close-master-pr.yml:15]: You've switched to '*our* repo,' but still have '*our* git usage documentation' linking to Space Wizards'.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

**Comment by DEATHB4DEFEAT** [comment]
> this should say Einstein's Engine

It was meant to be ambiguous so people don't need to edit it.

File: .github/workflows/close-master-pr.yml (hunk 1 above)
Comment (DEATHB4DEFEAT) [reply .github/workflows/close-master-pr.yml:15]:
```suggestion
        comment: "Thank you for contributing to our repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [the git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html). \n\n You can move your current work from the master branch to another branch by [branching](https://git-scm.com/docs/git-branch) from and [resetting](https://git-scm.com/docs/git-reset) the master branch."
```

File: .github/workflows/close-master-pr.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:    
     - uses: superbrothers/close-pull-request@v3
       with:
-        comment: "Thank you for contributing to the Space Station 14 repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [our git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html) \n\n You can move your current work from the master branch to another branch by doing `git branch <branch_name` and resetting the master branch."
+        comment: "Thank you for contributing to our repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [the git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html). \n\n You can move your current work from the master branch to another branch by [branching](https://git-scm.com/docs/git-branch) from and [resetting](https://git-scm.com/docs/git-reset) the master branch."
```
Comment (DangerRevolution) [inline .github/workflows/close-master-pr.yml:15]:
```suggestion
        comment: "Thank you for contributing to the Einstein Engines repository. Unfortunately, it looks like you submitted your pull request from the master branch. We suggest you follow [the git usage documentation](https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html). \n\n You can move your current work from the master branch to another branch by [branching](https://git-scm.com/docs/git-branch) from and [resetting](https://git-scm.com/docs/git-reset) the master branch."
```

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Review by FoxxoTrystan — COMMENTED** [review]
👍

**Review by Pspritechologist — APPROVED** [review]

**Review by DangerRevolution — APPROVED** [review]
Chat, clip it!

**Pull request merged**